src/coreclr/nativeaot/Runtime/SpinLock.h

```diff
@@ -3,8 +3,15 @@
 #ifndef __SPINLOCK_H__
 #define __SPINLOCK_H__
 
-// defined in gcrhenv.cpp
-bool __SwitchToThread(uint32_t dwSleepMSec, uint32_t dwSwitchCount);
+bool __SwitchToThread(uint32_t dwSleepMSec, uint32_t /*dwSwitchCount*/)
```

Comment:
This is header file. Does it need to be marked `inline`?

Reply:
It looks like this lock is not used anywhere. It is also a dangerously simplistic implementation that could only be ok in trivial never contending cases.  
Seems like better to just delete the file.